apps/web/src/components/member-avatar-name.tsx

```diff
@@ -15,10 +15,7 @@ type MemberAvatarNameProps = {
 export const MemberAvatarName: Component<MemberAvatarNameProps> = (props) => {
   return (
     <>
-      <MemberAvatar
-        member={props.member}
-        class={clsx(props.classes?.avatar, 'h-8 w-8 rounded-full border')}
-      />
+      <MemberAvatar member={props.member} class={clsx(props.classes?.avatar, 'h-8 w-8 rounded-full')} />
```

Comment:
**suggestion**

prefer `classList` usage over `clsx`

we could keep it in mind and change it iteratively